pmd-core/src/main/java/net/sourceforge/pmd/RuleSetParser.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd;
```

Comment:
Should we collect ruleset related APIs in a subpackage `net.sourceforge.pmd.ruleset`?